File: leetcode/1. Two Sum/main.go
package main

func twoSum(nums []int, target int) []int {
	var result []int

	for i := range nums {
		for j := i+1; j < len(nums); j++ {
			if nums[i] + nums[j] == target {
				result = []int{i, j}
				break
			}
		}
	}

	return result
}

File: leetcode/1. Two Sum/main.dart
class Solution {
  List<int> twoSum(List<int> nums, int target) {
    for (var i in range(nums.length)) {
      for (var j in range(nums.length)) {
        if (i != j && nums[i] + nums[j] == target) {
          return [i, j];
        }
      }   
    }

    return [];
  }

  Iterable<int> range(int end) sync* {
    for (int i = 0; i < end; i++) {
      yield i;
    }
  }
}

